Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -263,36 +261,7 @@ protected static function initializeExceptionStorage(Bootstrap $bootstrap): Thro
         }
 
         /** @var ThrowableStorageInterface $throwableStorage */
-        $throwableStorage = $storageClassName::createWithOptions($storageOptions);
-
-        $throwableStorage->setBacktraceRenderer(function ($backtrace) {
-            return Debugger::getBacktraceCode($backtrace, false, true);
-        });
-
-        $throwableStorage->setRequestInformationRenderer(function () {
-            $output = '';
-            if (!(Bootstrap::$staticObjectManager instanceof ObjectManagerInterface)) {
-                return $output;
-            }
-
-            $bootstrap = Bootstrap::$staticObjectManager->get(Bootstrap::class);
-            /* @var Bootstrap $bootstrap */
-            $requestHandler = $bootstrap->getActiveRequestHandler();
-            if (!$requestHandler instanceof HttpRequestHandlerInterface) {
-                return $output;
-            }
-
-            $request = $requestHandler->getHttpRequest();
-            $response = $requestHandler->getHttpResponse();
-            // TODO: Sensible error output
-            $output .= PHP_EOL . 'HTTP REQUEST:' . PHP_EOL . ($request ? '[request was empty]' : RequestInformationHelper::renderRequestHeaders($request)) . PHP_EOL;
-            $output .= PHP_EOL . 'HTTP RESPONSE:' . PHP_EOL . ($response ? '[response was empty]' : $response->getStatusCode()) . PHP_EOL;
-            $output .= PHP_EOL . 'PHP PROCESS:' . PHP_EOL . 'Inode: ' . getmyinode() . PHP_EOL . 'PID: ' . getmypid() . PHP_EOL . 'UID: ' . getmyuid() . PHP_EOL . 'GID: ' . getmygid() . PHP_EOL . 'User: ' . get_current_user() . PHP_EOL;
-
-            return $output;
-        });
-
-        return $throwableStorage;
```

Comment:
😱 all that is totally unnecessary?

Reply:
Its exactly the same code as in the constructor